### ISSUE TITLE
private should default to true during loaddata

### ISSUE THREAD

**Issue by lapo-luchini** [pinry/pinry#212]
When importing a dump from an older version which had no `private` field (but everything was always private), `false` is assumed and thus everything becomes visible.

Can be worked around like this:
`json_pp < db-backup.core.json| sed -re 's/"origin" : null/"private" : true/' > db-backup.core2.json`

**Comment by winkidney** [comment]
Maybe an option  in settings.py is a way to resolve this issue.(and most cases the workaround is enough haha).

**Comment by lapo-luchini** [comment]
True. Also, I'd like for "Add pin" to default to private pins as well (as IMvHO is better to exceed on the side of precaution)… which could use that setting as well.
(right now I'm using a locally-modified SPA client for that)
